malduck/string/ops.py

```diff
@@ -34,18 +36,17 @@ def asciiz(s):
     return s.split(b"\x00")[0]
 
 
-def chunks_iter(s, n):
+def chunks_iter(s: bytes, n: int) -> Iterator[bytes]:
```

Comment:
Could `str` be also passed here?
Generally I think it should work for any type that is iterable?

Reply:
Yes, this could be generic